src/espIdf/setTarget/index.ts

```diff
@@ -133,16 +137,30 @@ export async function setIdfTarget(placeHolderMsg: string, workspaceFolder: Work
           );
         }
 
-        await setTargetInIDF(workspaceFolder, selectedTarget);
+        const isTargetSet = await setTargetInIDF(
+          workspaceFolder,
+          selectedTarget
+        );
+        if (isTargetSet) {
+          const msg = l10n.t(
```

Comment:
This message and the output is still duplicate from `src/espIdf/setTarget/setTargetInIdf`. 

I suggest simplify this further, seems both codes could benefit for a single implementation